timedctl/helpers.py

```diff
@@ -69,6 +69,21 @@ def time_sum(arr):
     # format as HH:MM:SS
     return str(total)
 
+def activity_sum(arr):
+    """Sum up an array of activities."""
+    total = datetime.timedelta()
+    now = datetime.datetime.now()
+    for line in arr[1:]:
+        start = line[-2]
+        end = line[-1]
+        # if active activity, create end timestamp from `now`
+        if end == "active":
+            total += (now - datetime.datetime.strptime(now.strftime("%F")+"T"+start,"%Y-%m-%dT%H:%M:%S")) 
```

Comment:
`now.strftime("%F")+"T"+start` seems to (ab)use timezones for adding `start` to `now`

timedelta is a more pythonic  way to do this kind of math: `now + timedelta(hours=start)`